_includes/resources.html

```diff
@@ -1,14 +1,14 @@
-{% unless page.permalink == '/blog' %}
-  <a href="/blog">{% t home.blog.more %}</a>
+{% unless page.permalink == '/resources' %}
```

Comment:
I think we can remove the unless part and just always include the `more` link. If we did have permalinks (which we don't for resources per the frontmatter) I think we should still include the "more" yeah?

Reply:
Yes agree (I'm not sure what the original motivation was not to show "more" in that case, but if need be we can always add it back)